src/main/app/controller/courts/ContactsController.ts

```diff
@@ -45,13 +49,19 @@ export class ContactsController {
   }
 
   public async put(req: AuthedRequest, res: Response): Promise<void> {
-    const contacts = req.body.contacts as Contact[] ?? [];
-    contacts.forEach(c => c.fax = c.fax ?? false);
+    let contacts = req.body.contacts as Contact[] ?? [];
+    contacts.forEach(c => {
+      c.fax = c.fax ?? false;
```

Comment:
Is it better/safer to do the check here for whether c.fax is null, or is it better to make the value default to a false in the model class, or wherever it is initialised? 

In which case the only time where it would not be false, is where it is explicitly set to true.

Reply:
The fax property is used on a checkbox control. If that checkbox is checked, the fax property is true in the returned form. If that checkbox is unchecked, the property doesn't exist on the posted form.

Reply:
Also, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Does the browser compatiability matter here? Think it mentions some browsers are not compatiable with the ?? :O

Reply:
I don't think so - this is server-side code. Your link notes it'll work in node.js.